L1TriggerConfig/L1GtConfigProducers/interface/L1GtTriggerMenuConfigOnlineProd.h

```diff
@@ -262,7 +261,7 @@ Result L1GtTriggerMenuConfigOnlineProd::lexical_cast_from_hex(Source& value) con
 
   Result result;
   if (!(convertor >> std::hex >> result) || !convertor.eof()) {
-    throw boost::bad_lexical_cast();
+    throw std::exception();
```

Comment:
```suggestion
    throw throw std::runtime_error("bad lexical cast");
```
How about that instead?